community.license/pom.xml

```diff
@@ -11,7 +11,7 @@
     <groupId>org.kaazing</groupId>
     <artifactId>community.license</artifactId>
 
-    <name>Kaazing Corporation License</name>
+    <name>Community License</name>
     <url>https://github.com/kaazing/community</url>
```

Comment:
This is now being loaded via the classpath, so the fully qualified name is `org/kaazing/community/license/COPYRIGHT`.

A similar packaging strategy can be used for other use cases, with different package name preceding `COPYRIGHT`.

Note: the `COPYRIGHT` file is no longer written locally to a shared directory, so no need to add a suffix.